prometheus/0.9.1/artifacthub-pkg.yml

```diff
@@ -0,0 +1,41 @@
+version: 0.9.1
+name: prometheus-service
+displayName: Prometheus Service
+createdAt: '2022-08-30T07:17:28Z'
+description: Keptn service for utilizing Prometheus monitoring and alerting in Keptn
+logoURL: https://raw.githubusercontent.com/cncf/artwork/master/projects/prometheus/stacked/color/prometheus-stacked-color.svg
+digest: '2022-08-30T07:17:28Z'
+license: Apache-2.0
+homeURL: https://keptn.sh/docs/integrations/
+keywords:
+- keptn
+- prometheus
+- contrib
+- observability
+links:
+- name: Source
+  url: https://github.com/keptn-contrib/prometheus-service
+install: '## Execute the following steps to install prometheus-service:
+
+  ```bash helm install -n keptn prometheus-service https://github.com/keptn-contrib/prometheus-service/releases/download/<VERSION>/prometheus-service-<VERSION>.tgz
+
+  ### Install Role and RoleBinding to permit prometheus-service for performing operations
+  in the Prometheus installed namespace:
+
+  ```bash kubectl -n <PROMETHEUS_NS> apply -f https://raw.githubusercontent.com/keptn-contrib/prometheus-service/<VERSION>/deploy/role.yaml
```

Comment:
Role binding has been removed in [release 0.7.4](https://github.com/keptn-contrib/prometheus-service/releases/tag/0.7.4). This command can therefore be removed.